apps/web/components/App.tsx

```diff
@@ -0,0 +1,229 @@
+import {
+  BookOpenIcon,
+  DocumentTextIcon,
+  ExternalLinkIcon,
+  FlagIcon,
+  MailIcon,
+  ShieldCheckIcon,
+} from "@heroicons/react/outline";
+import { ChevronLeftIcon } from "@heroicons/react/solid";
+import Link from "next/link";
+import React from "react";
+
+import { useLocale } from "@lib/hooks/useLocale";
+
+//import NavTabs from "@components/NavTabs";
+import Shell from "@components/Shell";
+import Badge from "@components/ui/Badge";
+import Button from "@components/ui/Button";
+
+export default function App({
+  name,
+  logo,
+  body,
+  categories,
+  author,
+  price = 0,
+  commission,
+  type,
+  docs,
+  website,
+  email,
+  tos,
+  privacy,
+}: {
+  name: string;
+  logo: string;
+  body: React.ReactNode;
+  categories: string[];
+  author: string;
+  pro?: boolean;
+  price?: number;
+  commission?: number;
+  type?: "monthly" | "usage-based" | "one-time" | "free";
+  docs?: string;
+  website?: string;
+  email: string; // required
+  tos?: string;
+  privacy?: string;
+}) {
+  const { t } = useLocale();
+
+  /*const tabs = [
+    {
+      name: t("description"),
+      href: "?description",
+    },
+    {
+      name: t("features"),
+      href: "?features",
+    },
+    {
+      name: t("permissions"),
+      href: "?permissions",
+    },
+  ];*/
+
+  const priceInDollar = Intl.NumberFormat("en-US", {
+    style: "currency",
+    currency: "USD",
+    useGrouping: false,
+  }).format(price);
+
+  return (
+    <>
+      <Shell large>
+        <div className="-mx-8">
+          <div className="bg-gray-50 px-10">
+            <Link href="/apps">
+              <a className="mt-2 inline-flex px-1 py-2 text-sm text-gray-500 hover:bg-gray-100 hover:text-gray-800">
+                <ChevronLeftIcon className="h-5 w-5" /> {t("browse_apps")}
+              </a>
+            </Link>
+            <div className="flex items-center justify-between py-8">
+              <div className="flex">
+                <img className="h-16 w-16" src={logo} alt="" />
+                <header className="px-4 py-2">
+                  <h1 className="font-cal text-xl text-gray-900">{name}</h1>
+                  <h2 className="text-sm text-gray-500">
+                    <span className="capitalize">{categories[0]}</span> • {t("build_by", { author })}
+                  </h2>
+                </header>
+              </div>
+
+              <div className="text-right">
```

Comment:
Are we releasing this to production? If so what's the next step to add functionality to these buttons?